docker/nwchem/Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM        ubuntu:14.04
+
+LABEL maintainer="Muammar El khatib <melkhatibr@lbl.gov>, Chris Harris <chris.harris@kitware.com>"
+
+ENV         LC_ALL=C.UTF-8
+ENV         LANG=C.UTF-8
+ENV         NWCHEM_TOP="/opt/nwchem"
+
+RUN         apt-get update \
+            && apt-get -y upgrade \
+            && apt-get install -y gfortran libopenblas-dev libopenmpi-dev \
+                                  openmpi-bin tcsh make ssh patch curl git
+RUN         apt-get install -y software-properties-common \
+            && add-apt-repository -y ppa:ubuntu-toolchain-r/test \
+            && add-apt-repository ppa:jonathonf/python-3.6 \
```

Comment:
If we are able to bump the base ubuntu version, we may not need to add these external repos

Reply:
I agree. I think a good plan would be that I finish all the changes I can with this docker image, and then try to figure out how to upgrade from NWChem 6.6 to 6.8. That would make things easier. I am a Debian developer so I could just take the sources of NWChem from Debian or Ubuntu packages and apply the patch that makes the output to be in JSON format. Then we could host it somewhere and makes things easier. Does that make sense?